### PR TITLE
Add documentation to release Helm Chart

### PR DESCRIPTION
**Pull request by kezhenxu94**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: deploy/kubernetes/dolphinscheduler/Chart.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #
 
 apiVersion: v2
-name: dolphinscheduler
+name: dolphinscheduler-helm
```
Comment (kezhenxu94) [inline deploy/kubernetes/dolphinscheduler/Chart.yaml:19]: For release managers, if you plan to release Helm Chart for old versions, you need to modify this

**Review by SbloodyS — DISMISSED** [review]
+1

**Comment by zhongjiajie** [comment]
FYI, CI failed

**Review by caishunfeng — APPROVED** [review]
+1

**Pull request merged**